.travis.yml

```diff
@@ -8,12 +8,6 @@ dist: trusty
 
 install:
   - go get -t ./...
-  - go get golang.org/x/lint/golint
-  # Install gometalinter
-  - go get github.com/alecthomas/gometalinter
```

Comment:
This linter is deprecated and was causing errors in CI.  I tried the recommended replacement (golangci-lint), but it produced way too many errors.

Reply:
possibly look or file upstream as well? is this ults added cruft?

Reply:
Are you asking if this is the same as the upstream file?  If so, [yes](https://github.com/refraction-networking/utls/blob/master/.travis.yml)